src/modules/trade/pure/TradeNumberInput/index.tsx

```diff
@@ -106,11 +124,46 @@ export function TradeNumberInput(props: TradeNumberInputProps) {
             max={max}
             step={step}
           />
-          {showUpDownArrows && <InputArrows />}
+          {showUpDownArrows && <InputArrows onClickUp={onClickUp} onClickDown={onClickDown} />}
           {suffix && <Suffix>{suffix}</Suffix>}
         </InputWrapper>
       </>
     </TradeWidgetField>
   )
 }
-export { NumericalInput } from './styled'
+
+/**
+ * Increase `value` by `step`
+ *
+ * If no `value`, use `min`
+ * If no `min`, use `step`
+ *
+ * Uses BigNumberJS for avoiding JS finicky float point math
+ */
+function increaseValue(value: string, step: number, min: number | undefined): string {
+  const n = new BigNumberJs(value)
```

Comment:
Can we use native `BigInt` instead of `BigNumberJs`?

Reply:
The problem with BigInt is that it is, well, int.
I need float precision.
Of course could do it with BigInts but then we need aditional logic to transform the float values from and to decimals.